vulkan_generator/vulkan_utils/parsing_utils.py

```diff
@@ -51,10 +78,20 @@ def get_tail_from_tag(elem: ET.Element, tag: str) -> str:
     return value
 
 
-def try_get_tail_from_tag(elem: ET.Element, tag: str) -> Optional[str]:
+def try_get_tail_from_tag_in_children(elem: ET.Element, tag: str, recursive: bool = False) -> Optional[str]:
     """Tries to Gets the text of the element with the given tag
-    and returns None if the tag does not exits"""
-    for child in elem.iter():
+    and returns None if the tag does not exits
+
+    Args:
```

Comment:
Do we need to put this same large comment in so many places? In this case I think the name explains what the variable does without the need for any special documentation

Reply:
This one I put specifically because behaviour changes unexpectedly(due to how XML parser works)

Reply:
Let's not bloat the comments that much